e2e/input-box.e2e-spec.ts

```diff
@@ -194,13 +255,24 @@ describe('Input box', () => {
     });
```

Comment:
The `input-box-button-disabled-` screenshots are capturing the wrong div. I think you want ` expect('#input-box-button-disabled')...` instead of ` expect('#input-box-disabled')...`

Reply:
Fixed.